src/test/cli-integration/rbd/iscsi_client.t

```diff
@@ -9,7 +9,7 @@ Login to the target
 Make filesystem
 ===============
   $ device=`sudo multipath -l | grep 'LIO-ORG,TCMU device' | awk '{print $1}'`
-  > sudo mkfs.xfs /dev/mapper/$device -f | grep 'meta-data=/dev/mapper/mpath' | awk '{print $2}'
+  > sudo mkfs.xfs /dev/mapper/$device -f | grep 'meta-data=/dev/mapper' | awk '{print $2}'
```

Comment:
The commit message says "fix grep pattern" but it's not obivous (at least to me) why?